dist/index.html

```diff
@@ -1 +1,39 @@
-<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Leaderboard</title><script defer="defer" src="main.js"></script><link href="main.css" rel="stylesheet"></head><body><h1>Leaderboard</h1><div class="view-container"><div><div class="recent-scores"><h2>Recent scores</h2><button id="get-scores" type="button">Refresh</button></div><ul id="scores-container"></ul></div><div class="form-container"><h2>Add your score</h2><form id="add-score" action="#"><input id="input-name" placeholder="Your name" required> <input id="input-score" placeholder="Your score" required> <button id="submit-button" type="submit">Submit</button></form><div id="message"></div></div></div></body></html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.cdnfonts.com/css/ninja-naruto" rel="stylesheet">
+  <title>Leaderboard</title>
+<script defer src="main.js"></script><link href="main.css" rel="stylesheet"></head>
+
+<body>
+  <h1>Leaderboard</h1>
+  <div class="view-container">
+    <div>
+      <div class="recent-scores">
+        <h2>Recent scores</h2>
+        <button id="get-scores" type="button">Refresh</button>
+      </div>
+      <ul id="scores-container"> </ul>
+    </div>
+
+    <div class="form-container">
+      <h2>Add your score</h2>
+      <form id="add-score" action="#">
+        <input id="input-name" type="text" placeholder="Your name" required>
+        <input id="input-score" type="text" placeholder="Your score" required>
+        <button id="submit-button" type="submit">Submit</button>
+      </form>
+      <div id="message"></div>
+      <article>
+        <img src='https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/8932fa02-b339-4e8c-99d6-115e7cd29415/d8ll22q-c804667b-4306-424d-9235-d259ea5fb668.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7InBhdGgiOiJcL2ZcLzg5MzJmYTAyLWIzMzktNGU4Yy05OWQ2LTExNWU3Y2QyOTQxNVwvZDhsbDIycS1jODA0NjY3Yi00MzA2LTQyNGQtOTIzNS1kMjU5ZWE1ZmI2NjguanBnIn1dXSwiYXVkIjpbInVybjpzZXJ2aWNlOmZpbGUuZG93bmxvYWQiXX0.rjsTNN4j5XCLGC6RsbTyg_hQoJyfaR2_CHm2PMRpBgM' />
+        <img src='https://naruto-official.com/index/char_naruto.webp' />
+      </article>
+    </div>
```

Comment:
Your design
![image](https://github.com/mrcbq/Leaderboard/assets/78251927/a62b55e8-8db1-4296-b203-3a29b18b5b34)

Wireframe
![leaderboard_wireframe](https://github.com/mrcbq/Leaderboard/assets/78251927/e314c269-2102-40c5-ac07-f2a71e4f278b)

- [ ] To make our design layout look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), It would be nice if we remove the image shown above which is overlapping on the form inputs. 👀🛠👍

Reply:
It's done.